core/src/main/java/com/netflix/conductor/core/execution/WorkflowExecutor.java

```diff
@@ -345,6 +345,8 @@ public void completeWorkflow(Workflow wf) throws Exception {
         }
         Monitors.recordWorkflowCompletion(workflow.getWorkflowType(), workflow.getEndTime() - workflow.getStartTime(), wf.getOwnerApp());
         queueDAO.remove(deciderQueue, workflow.getWorkflowId());    //remove from the sweep queue
+      
```

Comment:
Can this be a configuration and runtime data deleted only if the configuration is set to that value ?
Conductor Config looks like a good place for it and it does not break the functionality where the users might be still using the runtime data from some other systems to query.

Reply:
Ok, However it is available from Elasticsearch.